mainsite/catalog/templates/catalog/update_actual_content.html

```diff
@@ -0,0 +1,262 @@
+{% comment %} Required additional vars to pass in: default_image_target,
```

Comment:
I think this should work if you have `{% load static %}` at the top, then `{% static 'images/imagenotfound.png' %}` for the img src.

Reply:
Okay, I swear that wasn't working earlier for some reason. Weird. Anyways, fixed.